pyomo/contrib/pynumero/src/CMakeLists.txt

```diff
@@ -139,27 +168,32 @@ IF( BUILD_ASL )
   ENDIF()
 ENDIF()
 
+#
+# build hsl interfaces
+#
 set(PYNUMERO_MA27_SOURCES
   "ma27Interface.cpp"
-  "ma27Interface.hpp"
 )
 
 IF( BUILD_MA27 )
   ADD_LIBRARY( pynumero_MA27 SHARED ${PYNUMERO_MA27_SOURCES} )
-  TARGET_LINK_LIBRARIES( pynumero_MA27 ${HSL_LIBRARY} )
+  IF( MA27_OBJECT )
+    TARGET_LINK_LIBRARIES( pynumero_MA27 ${MA27_OBJECT} )
+  ELSE()
+    TARGET_LINK_LIBRARIES( pynumero_MA27 ${MA27_LIBRARY} )
+  ENDIF()
   SET_TARGET_PROPERTIES( pynumero_MA27 PROPERTIES ENABLE_EXPORTS 1 )
```

Comment:
@jsiirola Do we need to add the following here (similar to what is done for ASL above)? If I am correct, let me know, and I'll add it in.

```
  if ( DL_LIBRARY )
    TARGET_LINK_LIBRARIES( pynumero_MA27 ${DL_LIBRARY} )
  ENDIF()
  TARGET_COMPILE_DEFINITIONS( pynumero_MA27 PRIVATE BUILDING_PYNUMERO_MA27 )
```

Reply:
Good question.  I don't think it will hurt, but I am a bit surprised that tests were passing.  Maybe it was because they were being skipped everywhere except Jenkins.  I will add it and see if tests clear up.